### PR TITLE
Categorize images

### PR DESCRIPTION
**Pull request by ErikBoesen**
The `images/` directory was kind of a mess, so I sorted them all into a few different directories.

### CODE REVIEW

File: assets/note.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+To anyone who finds this:
+It's probably okay to delete the `img` link in this directory. I only kept it here for backward compatibility since the /images directory was previously here and I didn't want to break any existing links. If you're reasing this enough time has passed that it probably doesn't matter anymore.
```
Comment (twinters007) [inline assets/note.txt]: reading*

File: index.html [hunk 2]
```diff
@@ -27,7 +27,7 @@ <h3>We are one of the most successful and inventive FIRST Robotics teams in the
 </div>
 <main>
     <div>
-        <img src="/assets/img/worlds2018.jpg" alt="The team at the 2018 FIRST World Championships, in front of the FIRST Logo at the Cobo Center in Detroit.">
+        <img src="/images/worlds2018.jpg" alt="The team at the 2018 FIRST World Championships, in front of the FIRST Logo at the Cobo Center in Detroit.">
         <p>We are team 1418 Vae Victis, a <a href="http://firstinspires.org/robotics/frc">FIRST Robotics Competition (FRC)</a> team from George Mason High School in Falls Church, VA. Since our founding in 2004, our team has grown in every way, and has maintained a high place in competition and in the community. To do this, we maintain an active and engaged student body of around fifteen students who work with the team on and off season.</p>
```
Comment (twinters007) [inline index.html]: should team be capitalized?

Comment (ianzullo) [reply index.html]: team isn't really a proper noun

**Comment by ErikBoesen** [comment]
Did I just make a merge commit?

Yeah, I did.

Darn.

**Comment by ErikBoesen** [comment]
I need another build season to set me right.

**Pull request merged**